integration/spark/spark3/src/main/java/io/openlineage/spark3/agent/lifecycle/plan/column/ExpressionDependencyCollector.java

```diff
@@ -105,7 +110,14 @@ static void collectFromNode(ColumnLevelLineageContext context, LogicalPlan node)
     if (node instanceof Project) {
       expressions.addAll(
           ScalaConversionUtils.<NamedExpression>fromSeq(((Project) node).projectList()));
-    } else if (node instanceof Aggregate) {
+    } else if (node instanceof CreateTableAsSelect
```

Comment:
That's the best workaround that comes to my mind without changing the behaviour for the any of the existing tests. Is this OK @tnazarew?  What do you say man?